x/ccv/provider/types/codec.go

```diff
@@ -40,6 +41,10 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 		(*sdk.Msg)(nil),
 		&MsgSubmitConsumerMisbehaviour{},
 	)
+	registry.RegisterInterface(
```

Comment:
Why is this needed?

Reply:
Good catch. It's not needed here. Removed in https://github.com/cosmos/interchain-security/pull/1223/